helm/values.yaml

```diff
@@ -12,12 +12,17 @@ serviceAccount:
   create: true
   name: ""
 
-## Enable authentication
-usePassword: false
+## Enable authentication (disabling till the backward compatibility is achieved)
+usePassword: true
 
 ## MongoDB admin password
 mongodbRootUsername: root
 mongodbRootPassword: ""
+mongoServiceUsername: "service-user-mongo"
+mongoServicePassword: ""
+mongoDeveloperUsername: "developer-user-mongo"
+mongoDeveloperPassword: ""
```

Comment:
Do we need to explicitly set these to empty string?
We could also skip them, right?

Also, can you update the comment in line # 18?